lib/linguist/heuristics.yml

```diff
@@ -295,6 +295,11 @@ disambiguations:
   - language: NL
     pattern: '^(b|g)[0-9]+ '
   - language: NewLisp
+- extensions: ['.odin']
+  rules:
+  - language: Odin
+    pattern: '(package\s+[\w0-9]+|import\s*"[\w0-9:/\.\.]+"|[\w0-9]+\s*::\s*proc\s*\(.*\)|[\w0-9]+\s*::\s*struct\s*|.*\/\/.*|for\s+.*in\s+.*)'
+  - language: Object Data Instance Notation
```

Comment:
A better heuristic would be:

~~~yml
rules:
- language: Object Data Instance Notation
  pattern: '(?:^|<)\s*[A-Za-z0-9_]+\s*=\s*<'
- language: Odin
~~~

OpenEHR's ODIN has a very predictable and distinctive syntax:

~~~object-data-instance-notation
term = <text = <"plan">; description = <"The clinician's advice">>
term = <text = <"plan"> description = <"The clinician's advice">>
term = <
	text = <"plan">
	description = <"The clinician's advice">
>
~~~

Provided none of the above lines are also valid in Odin programs, the heuristic I've suggested should be fine.

Reply:
Sounds good 👍